packages/jamf-adapter/src/definitions/fetch/transforms/utils.ts

```diff
@@ -79,3 +87,23 @@ export const removeSelfServiceSecurityPassword = (value: Record<string, unknown>
     }
   }
 }
+
+export const maskPayloadsPassword = (value: Record<string, unknown>): void => {
```

Comment:
shouldn't this be (same for `maskPasswordsForScriptsObjectArray`) ?
```suggestion
export const maskPayloadsPassword = (value: Values): void => {
```

Reply:
(I've changed in all the original functions in this page as well)